dashboards/resources.libsonnet

```diff
@@ -213,8 +220,17 @@ local g = import 'grafana-builder/grafana.libsonnet';
         g.row('Memory Usage')
         .addPanel(
           g.panel('Memory Usage') +
-          g.queryPanel('sum(container_memory_usage_bytes{namespace="$namespace", pod_name="$pod", container_name!="POD", container_name!=""}) by (container_name)', '{{container_name}}') +
```

Comment:
I think this should continue to use the full memory usage, for more advanced troubleshooting I feel users are going to either do their own queries or drill down where the pod dashboard does show the difference.

Reply:
Sorry this was on the wrong line, I meant to comment on the namespace dashboard.

Reply:
Ok, will change that.